### ISSUE TITLE
Post summary at end to configurable slack channel

### ISSUE THREAD

**Issue by supremebeing7** [MammothHR/release_robot#7]

**Comment by henrebotha** [comment]
@supremebeing7 At a glance it looks like `/lib/release_robot.rb:28` would be the right place to insert this, is that correct?

Do you want the same output (i.e. `ReleaseRobot::Printer#print_all`) sent to Slack, or something different?

**Comment by supremebeing7** [comment]
Yeah. In the current state of the code, that seems like a reasonable place to put this.

I think probably the same output should go to slack.

________________________________
